src/browser/provider/built-in/dedicated/chrome/cdp-client/index.ts

```diff
@@ -248,7 +249,7 @@ export class BrowserClient {
 
                 if (this._proxyless) {
                     await this._injectProxylessStuff(client);
-                    this._clientFunctionExecutor.setupFramesWatching(client.Runtime);
+                    ExecutionContext.initialize(client.Runtime, client.Page);
```

Comment:
I guess we reduce the parameters count for the `ExecutionContext.initialize` function.
```js
ExecutionContext.initialize(client);
```